command/cp.go

```diff
@@ -64,6 +64,12 @@ Examples:
 
 	10. Mirror an S3 prefix to target S3 prefix
 		 > s5cmd {{.HelpName}} -n -s -u s3://bucket/source-prefix/* s3://bucket/target-prefix/
+
+	11. Perform KMS Server Side Encryption of the object(s) at the destination
+		> s5cmd -sse aws:kms {{.HelpName}} s3://bucket/object s3://target-bucket/prefix/object
```

Comment:
-sse flag is part of the subcommand, not the globals. this command is gonna fail.